ppr-ui/tests/unit/test-data/mock-new-registration.ts

```diff
@@ -38,6 +38,12 @@ export const mockedAddress1: AddressIF = {
 
 export const mockedGeneralCollateral1: string = 'TEST1 GENERAL COLLATERAL'
 
+export const mockedGeneralCollateral2: Array<GeneralCollateralIF> = [
+  {
```

Comment:
For this, do we need it to be an array? I changed it to a single string

Reply:
For the API new financing statement call it is an array. In the UI it is a single element. The setup for the API call takes care of the mapping from UI to API.